serverless.yml

```
@@ -0,0 +1,20 @@
+service: node-canvas-lambda
+
+provider:
+  name: aws
+  runtime: nodejs14.x
```

ooops this needs to be 16
```suggestion
  runtime: nodejs16.x
```

---

Done.

---

perfect thanks :)